zenodo/modules/deposit/jsonschemas/deposits/records/record-v1.0.0.json

```diff
@@ -1,16 +1,839 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
+  "additionalProperties": false,
```

Comment:
Is it possible to add some sort of warning in the description or similar that this is an auto-generated schema that should not be modified. I know we publish it so the message should be gentle, but important that developers don't make any edits directly in the file.